rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -10,6 +13,24 @@
 | 30 GB of disk space | 30 GB (SSD) of disk space
 |===
 
+To run more than one process in parallel, when using Secure Sessions, you must add 2 GB of RAM plus the RAM needed to run the process for each parallel run to these recommended values.
+
+For example, to run two different processes in parallel:
+
+* Process A requires one GB of RAM and runs 10 times in parallel.
+* Process 2 requires three GB of RAM and runs five times in parallel.
```

Comment:
```suggestion
* Process B requires three GB of RAM and runs five times in parallel.
```

Reply:
But now I see the problem with naming them alphabetically, it doesn't work with the P__n_ in the formula. 😭